src/Insect/Parser.purs

```diff
@@ -261,6 +261,7 @@ imperialUnitDict = Dictionary
   , Q.inch ==> ["inches", "inch", "in"]
   , Q.yard ==> ["yards", "yard", "yd"]
   , Q.foot ==> ["feet", "foot", "ft"]
+  , Q.thou ==> ["thou", "mils", "mil"]
```

Comment:
Do we really need the two alternative versions (`mils` and `mil`)? I have personally never heard of them and I think they could be confused with `mile`/`miles`. Admittedly, I have never heard of `thou` either :smile:.

Reply:
I come across `mil` much more often than `thou`. But I don't mind just remembering to use `thou`. Will update.

(Yes they're silly units, but unfortunately still in use!)